### DISCUSSION TITLE
Customized Streamlit app

### DISCUSSION THREAD

**Discussion by Muhammad-Ali-Butt** [AammarTufail/python_ka_chilla2023#108] [Show and tell]
https://muhammad-ali-butt-streamlit-apps-stlitapp-agbi1p.streamlit.app/
Theme changed for streamlit app, just look up streamlit config.toml and you can do it too and i also added an animation as gif. To add custom theme create a new file as txt in same folder in which you have your streamlit environment files etc and rename it to config plus change extension to .toml i.e. config.toml and then simply copy paste the following in the config.toml with which you can play as you wish:

[theme] 
primaryColor="gainsboro"
backgroundColor= "darkblue"  
secondaryBackgroundColor="#00203FFF"
textColor="#ffffff"
font="sans serif"

**There are a lot of other different things as well which you can add in config file as per your wish**

**Comment by zohaib-qureshi** [comment]
This is great, thanks for sharing!

**Reply by Muhammad-Ali-Butt** [reply]
My pleasure, glad you liked it zohaib bhai 😊.
